xAPI.md

```diff
@@ -600,12 +600,18 @@ the resulting document stored in the LRS is:
 
 ## 2.0 Statements  
 
-### 2.1 Purpose
-
 ###### Description 
 The Statement is the core of the xAPI. All learning events are stored as Statements.
 A Statement is akin to a sentence of the form "I did this".
 
+### 2.1 Purpose
+
+Statements are the evidence for any sort of experience or event which is to be tracked in xAPI. Statements can be expressed 
```

Comment:
That doesn't say the meaning being discussed, which is that it is possible to write a readable natural language sentence that matches fairly reasonably the content of the Statement (but is not exactly the content in the Statement).